tests/convert_v3_to_v4_test.py

```diff
@@ -0,0 +1,49 @@
+import os
+from pathlib import Path
+
+from floris import FlorisModel
+
+
+CONVERT_FOLDER = Path(__file__).resolve().parent / "v3_to_v4_convert_test"
+FLORIS_FOLDER = Path(__file__).resolve().parent / ".." / "floris"
+
+
+def test_v3_to_v4_convert():
+    # Note certain filenames
+    filename_v3_floris = "gch.yaml"
+    filename_v4_floris = "gch_v4.yaml"
+    filename_v3_turbine = "nrel_5MW_v3.yaml"
+    filename_v4_turbine = "nrel_5MW_v3_v4.yaml"
+
+    # Copy convert scripts from FLORIS_FOLDER to CONVERT_FOLDER
+    os.system(f"cp {FLORIS_FOLDER / 'convert_turbine_v3_to_v4.py'} {CONVERT_FOLDER}")
+    os.system(f"cp {FLORIS_FOLDER / 'convert_floris_input_v3_to_v4.py'} {CONVERT_FOLDER}")
+
+    # Change directory to the test folder
+    os.chdir(CONVERT_FOLDER)
+
+    # Print the current directory
+    print(os.getcwd())
+
+    # Run the converter on the turbine file
+    os.system(f"python convert_turbine_v3_to_v4.py {filename_v3_turbine}")
+
+    # Run the converter on the floris file
+    os.system(f"python convert_floris_input_v3_to_v4.py {filename_v3_floris}")
+
+    # Go through the file filename_v4_turbine and where the place-holder string "XXXXX" is found
+    # replace it with the string f"!include {filename_v4_turbine}"
+    with open(filename_v4_floris, "r") as file:
+        filedata = file.read()
+    filedata = filedata.replace("XXXXX", f"!include {filename_v4_turbine}")
+    with open(filename_v4_floris, "w") as file:
+        file.write(filedata)
+
+    # Now confirm that the converted file can be loaded by FLORIS
+    _ = FlorisModel(filename_v4_floris)
```

Comment:
I think we should also confirm that it runs, because I'm not 100% sure that just loading it catches all possible missing fields? I don't think we should `set` anything, just instantiate to `fmodel` and `run`.

Reply:
Good suggestion!  Adding now